components/Chat/ChatSettings/ChatSettingsPanel/ChatGeneralSettingsPanel.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <v-card
+    flat
+    tile
+    class="pb-3 px-3"
+  >
+    <!-- General Settings Toggles -->
+    <div>
+
+      <!-- Global Chat -->
+      <v-switch
+        v-model="globalChat"
+        label="Global Chat"
+        color="primary"
+        hide-details
+        dense
+        inset
+      />
+
+      <!-- Receive @'s in Local -->
+      <v-switch
+        v-model="receiveMentionsInLocal"
+        :disabled="globalChat"
+        label="Recieve @'s while in local"
```

Comment:
Spelling seems wierd :)

Reply:
Apparently 'leave review' and 'leave single comment' do the same thing

Reply:
fixed. In my defense, you made that label.